test/cli/test_login.rb

```diff
@@ -0,0 +1,237 @@
+require_relative "test_helper"
```

Comment:
Add `# frozen_string_literal: true` to the beginning of this file (it's good practice in Ruby 2.3+).